### ISSUE TITLE
Unicode strings break layout

### ISSUE THREAD

**Issue by honzajde** [alexhallam/tv#154]
Unicode strings break layout. 

I have tried `xsv table` for comparison that handles it well.

**Comment by alexhallam** [comment]
Can I see some examples. I do use a library that is supposed to handle unicodes, but it must be falling short if you are experiencing this issue.

**Comment by honzajde** [comment]
```csv
name,count
太吾绘卷,123
A line,123
```

```sh
        tv dim: 2 x 2
        name   count 
     1  太吾绘卷   123   
     2  A line 123  
```

**Comment by alexhallam** [comment]
Makes sense. I am away on a work trip the rest of the week I will follow up next week.

**Comment by Yomguithereal** [comment]
If I recall correctly, you need to consider both graphemes and unicode width to be able to do this correctly, and this has some perf cost. Is this possible to help you on this somehow? Should I try a PR?

**Comment by alexhallam** [comment]
Please do submit a PR

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by zy6p** [comment]
May I kindly ask if there has been any progress so far?
